### PR TITLE
add param 'values' in cce addon

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
add param 'values' in cce addon

### CODE REVIEW

File: docs/resources/cce_addon.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ The following arguments are supported:
 * `cluster_id` - (Required, String, ForceNew) ID of the cluster. Changing this parameter will create a new resource.
 * `template_name` - (Required, String, ForceNew) Name of the addon template. Changing this parameter will create a new resource.
 * `version` - (Required, String, ForceNew) Version of the addon. Changing this parameter will create a new resource.
+* `values` - (Optional, Map, ForceNew) Add-on template installation parameters. These parameters vary depending on the add-on.
```
Comment (niuzhenguo) [inline docs/resources/cce_addon.md]: values is List instead of Map

File: huaweicloud/resource_huaweicloud_cce_addon_v3.go [hunk 2]
```diff
@@ -53,10 +53,49 @@ func resourceCCEAddonV3() *schema.Resource {
...
+				},
+			},
 		},
 	}
 }
 
+func getValuesValues(d *schema.ResourceData) (basic, custom map[string]interface{}, err error) {
+	values := d.Get("values").([]interface{})
+	if len(values) == 0 {
+		log.Printf("[WARNING] no values are set for CCE addon")
```
Comment (niuzhenguo) [inline huaweicloud/resource_huaweicloud_cce_addon_v3.go]: Seems we don't need a WARNING here as values are commonly empty

File: docs/resources/cce_addon.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/cce_addon.md]: fixed

File: huaweicloud/resource_huaweicloud_cce_addon_v3.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_addon_v3.go]: fixed

**Pull request merged**